src/index.ts

```diff
@@ -181,12 +181,18 @@ class Monika extends Command {
     }),
 
     force: flags.boolean({
-      description: 'force command',
+      description: 'Force command',
       default: false,
     }),
 
     'status-notification': flags.string({
-      description: 'cron syntax for status notification schedule',
+      description: 'Cron syntax for status notification schedule',
+    }),
+
+    notification: flags.string({
+      description:
+        'Override notification(s) provided by config (-c/--config) with a separate file',
+      dependsOn: ['config'],
```

Comment:
CMIIW, with `dependsOn: ['config']` here, user won't be able to execute `monika --notification <path to notif>` and use `monika.json` file in the current directory right?

If it is as I thought, no need to use dependsOn here.

Reply:
I tested this and the dependency declaration does not affect that `--config`'s default behaviour